src/main/java/com/darkyen/tproll/util/RenderedMarker.java

```diff
@@ -0,0 +1,6 @@
+package com.darkyen.tproll.util;
+
+/**
+ *
+ */
+public abstract class RenderedMarker extends SimpleMarker {}
```

Comment:
Would be better as interface extending Marker and a method that takes `StringBuilder` for the actual appending (appending `Marker.getName()` is not ideal, but can be the default). And some documentation.